### PR TITLE
ingenia motor example

### PR DESCRIPTION
**Pull request by trns1997**

### CODE REVIEW

File: examples/ingenia_control/include/IngeniaProtocol.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        constexpr uint32_t TPDOMappingCount = sizeof(TPDOMapping) / sizeof(uint32_t);
+
+        struct Output
+        {
+            uint16_t controlWord;     // 0x2010
+            uint16_t modeOfOperation; // 0x2014
+            float targetTorque;       // 0x2022 Nm
+            float maxCurrent;         // 0x21E0 A
+            int32_t targetPosition;   // 0x2020 ticks
+            // Possible future registers:
```
Comment (Siviuze) [inline examples/ingenia_control/include/IngeniaProtocol.h:39]: commented code

File: examples/ingenia_control/include/IngeniaProtocol.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+namespace kickcat
+{
+    namespace pdo
+    {
+        /// \details See CANOpen DS402 document for description of fields.
+        ///          Here fields are name "Torque" but actually refer to current in amps.
+        struct Input
+        {
+            uint16_t statusWord;             // 0x2011
```
Comment (Siviuze) [inline examples/ingenia_control/include/IngeniaProtocol.h:12]: snake case

File: examples/ingenia_control/include/IngeniaProtocol.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            int32_t actualPosition;          // 0x2030 ticks
+            float actualVelocity;            // 0x2031 rev/s
+            float demandTorque;              // 0x2053 Nm
+            float actualTorque;              // 0x2029 Nm
+            float dcVoltage;                 // 0x2060 V
+            int32_t demandPosition;          // 0x2078 ticks
+            int32_t auxiliarPosition;        // 0x2033
+            uint16_t imuAccelX;              // 0x218A ADC steps (to be converted)
```
Comment (Siviuze) [inline examples/ingenia_control/include/IngeniaProtocol.h:20]: seems to be related to a custom hardware

Comment (trns1997) [reply examples/ingenia_control/include/IngeniaProtocol.h:20]: I will remove it then

**Pull request merged**